### ISSUE TITLE
Switch from Yarn to npm

### ISSUE THREAD

**Issue by katydecorah** [mapbox/mapbox-gl-js-docs#472]
When we first split the docs from mapbox-gl-js repo, we attempted to also switch from yarn to npm (which is standard for docs sites). Due to an unidentified bug that caused the build to fail, we weren't able to make the switch so we stuck with Yarn. 

We identified and fixed the bug (https://github.com/mapbox/mapbox-gl-js-docs/pull/60#issuecomment-739027243) and can now safely move over to npm.

@mapbox/gl-js @mapbox/docs Any objections to switching from yarn to npm for this repository?

**Comment by HeyStenson** [comment]
Thanks @katydecorah! Switching to npm is going to make the Mapbox docs contributor experience ✨ so cohesive! ✨

**Comment by ryanhamley** [comment]
Sounds good! Makes me wonder if GL JS should do this too

**Comment by mourner** [comment]
Yeah, at the time we made the decision to switch to Yarn, we had struggled with NPM so much with its tons of bugs, abysmal performance and lack of reproducible installs. However, in recent years, NPM has caught up quite a bit, introducing package locks, improving performance dramatically, and fixing a lot of issues that has plagued us, so we should consider switching back. More so now that Yarn v1 isn't being developed, and v2 likely won't fit our use case.
